packages/webapp/src/containers/Finances/NewExpense/AddExpense/index.jsx

```diff
@@ -200,10 +205,11 @@ class AddExpense extends Component {
                     ))}
                   </div>
                   <div className={styles.addContainer}>
-                    <div className={styles.greenPlus}>+</div>
-                    <button onClick={() => this.addSubExpense(k)}>
-                      {this.props.t('EXPENSE.ADD_MORE_ITEMS')}
-                    </button>
+                    <Tooltip title={this.props.t('EXPENSE.ADD_MORE_ITEMS')}>
+                      <div className={styles.greenPlus} onClick={() => this.addSubExpense(k)}>
+                        +
```

Comment:
The text no longer appears on the page-- just as the tool tip. I don't think the ticket said anything about it. Was it intentional?

Reply:
Yes it was intentional, no the ticket doesn't say that, I just thought the text looked odd and a tooltip would work better. I should have consulted with Sasha first.